jest-test-mode.el

```diff
@@ -198,6 +210,15 @@ mode"
       (when-let ((example (match-string 1 text)))
         (substring example 1 -1)))))
 
+(defun jest-test-unit-at-point ()
+  "Find the topmost it or test block from where the cursor is and extract the name."
+  (save-excursion
+    (re-search-backward "\\(it\([\'\"]\\|test\([\'\"]\\)")
+    (let ((text (thing-at-point 'line t)))
+      (string-match "\\(it\(\\|test\(\\)\\(.*\\)," text)
```

Comment:
suggestion: Can the search also include `describe` -- if the cursor is at the point of a nested describe block, I would expect this test function to pick it up.